### ISSUE TITLE
All notices should use a spokenMessage

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#9425]
The failure notices in `packages/editor/src/store/effects/posts.js` should also use a `spokenMessage` to produce an audible message in the ARIA live region. This should be pretty standard for all the notices and it's already implemented in some of them, for example the success notices.

Also, the string used for the spoken message should not include links. See for example 
https://github.com/WordPress/gutenberg/blob/a731af5223b6a00021cb8f20e71df61ab1e634a0/packages/editor/src/store/effects/posts.js#L219-L227

**Comment by afercia** [comment]
Ah OK I understand better now. So, by default the audible message will be the string used for the visible message. Only if a different audible message is desired, then it's possible to use `spokenMessage` to differentiate it:

https://github.com/WordPress/gutenberg/blob/7462c6388df3780cfe3b74ba43f932660b926b64/packages/editor/src/store/effects.js#L212-L215

**Comment by aduth** [comment]
The behavior was changed in the introduction of the notices module in #9617 to make the displayed content align to the spoken text, by limiting the acceptable content of a notice to plaintext.

**Comment by gziolo** [comment]
@afercia is it still an issue?

**Comment by nerrad** [comment]
The refactor in #13716 is mostly preserving the existing behaviour in the code existing in master now.  As @aduth mentioned, #9617 seems to be what would have had the most impact on this issue.  So awaiting confirmation from @afercia, it looks to me like this issue shouldn't exist now?

**Comment by afercia** [comment]
To my understanding, all notices have now built-in ability to `speak()` their message (as long as the default option `speak: true` is not changed) so I guess this issue can be closed.
